stages/test.md

```diff
@@ -6,15 +6,15 @@
 ### __Stage description:__  
 This practical task consists of constructing a RACIVS matrix, this project management technique will be used to document and clearly outline the important steps that occur in a DevOps Testing team. The importance of the Testing team is to streamline the process which allows for continuous integration and deployment of program code.
 
-| Pipeline Stage:<br>Test  | Developer  | Test Lead  | Client  | Project Manager  | Tester  |Sys Admin  |
-|----------------------------- |-------- |-------- |-------- |-------- |-------- |-------- |
-| Automate Unit Test Execution + Reporting                     |         |         |         |         |         |         |
-| Resolve Failed Tests                      |         |         |         |         |         |         |
-| Write SIT Test Plan / Scripts                       |         |         |         |         |         |         |
-| Execute SIT                      |         |         |         |         |         |         |
-| Write UAT Test Plan / Scripts                       |         |         |         |         |         |         |
-| Execute UAT                      |         |         |         |         |         |         |
-| Compile Test Completion Report                       |         |         |         |         |         |         |
+| Pipeline Stage:<br>Test                   |   Developer  | Test Lead  | Client  |  Project Manager  | Tester  |Sys Admin  |
+|------------------------------------------ |------------- |----------- |-------- |------------------ |-------- |---------- |
+| Automate Unit Test Execution + Reporting  |      I       |     RA     |         |        C          |    I    |           |
+| Resolve Failed Tests                      |      R       |     A      |         |        S          |    V    |           |
+| Write SIT Test Plan / Scripts             |      I       |     RA     |         |        S          |         |           |
+| Execute SIT                               |      R       |     V      |         |        I          |    R    |           |
+| Write UAT Test Plan / Scripts             |              |            |         |                   |         |           |
+| Execute UAT                               |              |            |    R    |                   |         |           |
+| Compile Test Completion Report            |              |     V      |    I    |        S          |     I   |      R    |
   
```

Comment:
1. Tester - R
2. Test Lead - AV
3. PM + Client - S
4. Dev - I
5. Sys Admin - I (so can be pushed to next phase?)